recipes/perl-xml-parser/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "2.44_01" %}
+
+package:
+  name: perl-xml-parser
+  version: {{ version }}
+
+source:
+  fn: XML-Parser-{{ version }}.tar.gz
+  url: https://cpan.metacpan.org/authors/id/T/TO/TODDR/XML-Parser-{{ version }}.tar.gz
+  sha256: 5310ea5c8c707f387589bba8934ab9112463a452f828adf2755792d968b9ac7e
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - perl 5.22.2.1
```

Comment:
@bgruening that's exactly where I got my inspiration from (hence asking you for guidance)!

Removing the expat dependency was just another desperate attempt after I've tried all combinations of perl + expat but got [CI failures on OSX](https://travis-ci.org/conda-forge/staged-recipes/builds/253990897).